pkg/kubelet/pod/pod_manager.go

```diff
@@ -77,10 +78,14 @@ type Manager interface {
 	// this means deleting the mappings related to mirror pods.  For non-
 	// mirror pods, this means deleting from indexes for all non-mirror pods.
 	DeletePod(pod *v1.Pod)
-	// DeleteOrphanedMirrorPods deletes all mirror pods which do not have
-	// associated static pods. This method sends deletion requests to the API
-	// server, but does NOT modify the internal pod storage in basicManager.
-	DeleteOrphanedMirrorPods()
+	// GetOrphanedMirrorPodNames returns names of orphaned mirror pods
+	GetOrphanedMirrorPodNames() []string
+	// GetPendingPodUID returns UID for static pods pending termination
+	GetPendingPodUID(podFullName string) (kubetypes.UIDWithGracePeriod, bool)
+	// IsPodPendingExit checks whether the UID belongs to pending pod
+	IsPodPendingExit(uid types.UID) bool
+	// DeletePendingPod removes map entry for static pod pending termination
+	DeletePendingPod(podFullName string)
```

Comment:
same comment here, this doesn't appear to specific to static pods.

Reply:
Renamed to DeletePodPendingTermination